src/popup/router/components/Modals/AccountDetails.vue

```diff
@@ -355,6 +385,45 @@ export default {
         }
       }
     }
+
+    .names-list-wrapper {
+      padding: 0;
+
+      ::v-deep .name-item {
+        background-color: variables.$color-bg-4;
+        word-break: normal;
+
+        .container {
+          height: auto;
+        }
+
+        .expand {
+          text-align: left;
+        }
+      }
+
+      .tabs {
+        padding: 6px;
+        gap: 16px;
+
+        @include mixins.flex(flex-start, center, row);
+
+        .button-plain {
```

Comment:
I suggest to separate the concerns. Parent component should not use the child component's classes as those might change.